torch/_prims/nvfuser_executor.py

```diff
@@ -66,8 +66,8 @@ def make_nvfuser_fusion(gm: GraphModule, *nv_args_templates):
                 f"Node {node} with target {node.target} does not support nvfuser"
             )
 
-    fusion = Fusion()
-    with FusionDefinition(fusion) as fd:
+    fusion_manager = FusionManager.get()
+    with FusionDefinition(fusion_manager) as fd:
```

Comment:
Nit: If `FusionManager` is a singleton, then would it make sense to just allow `FusionDefinition` to omit its argument? Something like:
```python
with FusionDefinition as fd:
```

Reply:
I think you are right.  I am going to refactor this PR to not use the `FusionManager` as an argument.  I think I am also going to rename it `FusionCache`.

Reply:
I removed the singleton in favor reverting to the old behavior where a fusion is passed into the `FusionDefinition`.   Then you can execute the fusion.